test/errorHandling.js

```diff
@@ -125,8 +138,9 @@ describe('#errorHandling', () => {
 
         statsd.send('test title', [], error => {
           assert.ok(error);
-          assert.equal(error.code, 'ENOTFOUND');
+          assert.equal(error.code, serverType === 'uds' ? '-2' : 'ENOTFOUND');
```

Comment:
Change in error.code, one of the few actual differences when using uds in these tests